ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml

```diff
@@ -1,25 +1,35 @@
 base_images:
   base:
-    name: "4.9"
+    name: ubi
```

Comment:
The other examples I have at hand use 

```
  base:
    name: "4.10"
    namespace: ocp
    tag: base
```

Reply:
I've mostly looked at node-label-operator and it's using the same base:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-kni/node-label-operator/openshift-kni-node-label-operator-main.yaml#L2. I also checked its Dockerfile: https://github.com/openshift-kni/node-label-operator/blob/main/Dockerfile#L2 which is the same as to what cluster-group-upgrades is using: https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/Dockerfile#L2.

Please let me know what you think.

Reply:
From my POV, if using this image works for @irinamihai , I have no real concerns.